charts/cmc-claim-store/values.aat.template.yaml

```diff
@@ -5,10 +5,6 @@ java:
     PDF_SERVICE_URL: http://cmc-pdf-service-java.software-engineering
     SEND_LETTER_URL: http://rpe-send-letter-service-java.software-engineering
     DOCUMENT_MANAGEMENT_URL: http://dm-store-java.ccd
-    PAY_URL: false
-    FEES_URL: false
```

Comment:
Are you sure you want to remove these defaults from here?

Reply:
yes ... it has been already set to false in *values.yaml* ... so no need to duplicate it there

Reply:
👍